dotCMS/src/main/java/com/dotcms/publisher/integrity/IntegrityDataGeneratorThread.java

```diff
@@ -53,8 +55,6 @@ public void run() {
                 HibernateUtil.closeSession();
```

Comment:
shouldn't be removed this one HibernateUtil.closeSession(); too?

Reply:
only leave `servletContext.setAttribute("integrityDataGenerationStatus", ProcessStatus.FINISHED);` in the finally